src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/world/scaffold/ModuleScaffold.kt

```diff
@@ -175,10 +229,51 @@ object ModuleScaffold : Module("Scaffold", Category.WORLD) {
         SilentHotbar.resetSlot(this)
     }
 
+    private fun findRotation(target: BlockPlacementTarget?): Rotation? {
```

Comment:
> this function should never be called if the godbridge aim mode is not activated

My idea with the function was that it handled all custom rotation, so if new modes would be added they would be handled inside of the function as well.

> Can this function be turned into a FaceTargetPositionFactory based on CenterTargetPositionFactory? That would be the most elegant way to add this functionality

Yes, that would definitely be the most elegant and cleanest way, but it wouldn't really make sense to convert the rotation back to a position just to convert it back to a rotation again. I think it should be a TargetRotationFactory instead. This would make more sense, and it would enable modes to elegantly add custom rotation when needed (I'm thinking about things like telly where you want it to point in a specific direction when not placing blocks)

I believe that the correct way of implementing it would be to take an approach such as the one I used with the target renderer.